### PR TITLE
dockershim: clean up comments

### PR DESCRIPTION
**Pull request by yujuhong**
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/36664)

### CODE REVIEW

File: pkg/kubelet/dockershim/docker_container.go [hunk 1]
```diff
@@ -79,7 +79,8 @@ func (ds *dockerService) ListContainers(filter *runtimeApi.ContainerFilter) ([]*
 }
 
 // CreateContainer creates a new container in the given PodSandbox
-// Note: docker doesn't use LogPath yet.
+// Note: `LogPath` is not used because docker cannot store the log to an
```
Comment (Random-Liu) [inline pkg/kubelet/dockershim/docker_container.go]: We are using `LogPath` right? We just make symlink, instead of creating actual log file.

**Review by Random-Liu — COMMENTED** [review]
LGTM with on nit

File: pkg/kubelet/dockershim/docker_container.go [hunk 2]
```diff
@@ -79,7 +79,8 @@ func (ds *dockerService) ListContainers(filter *runtimeApi.ContainerFilter) ([]*
 }
 
 // CreateContainer creates a new container in the given PodSandbox
-// Note: docker doesn't use LogPath yet.
+// Docker cannot store the log to an arbitrary location (yet), so we create an
+// symlink to at LogPath, linking to the actual path of the log.
```
Comment (Random-Liu) [inline pkg/kubelet/dockershim/docker_container.go]: `to at` seems weird to me.

Comment (yujuhong) [inline pkg/kubelet/dockershim/docker_container.go]: fixed

**Comment by Random-Liu** [comment]
LGTM

**Pull request merged**